lib/Controller/WorkspaceController.php

```diff
@@ -188,19 +188,20 @@ public function findAll(): JSONResponse {
 		$workspaces = $this->workspaceService->getAll();
 		$spaces = [];
 		foreach ($workspaces as $workspace) {
-			$space = array_merge(
-				$this->folderHelper->getFolder(
-					$workspace['groupfolder_id'],
-					$this->rootFolder->getRootFolderStorageId()
-				),
-				$workspace
+			$folderInfo = $this->folderHelper->getFolder(
+				$workspace['groupfolder_id'],
+				$this->rootFolder->getRootFolderStorageId()
 			);
+			$space = ($folderInfo !== false) ? array_merge(
+				$folderInfo,
+				$workspace
+			): $workspace;
 
-			$gids = array_keys($space['groups']);
+			$gids = array_keys($space['groups'] ?? []);
```

Comment:
It's very strange :thinking: 

Normally, you must have the key groups.
Could you explain your case to me so that I can understand the problem in general?

Reply:
Seems i've already explained. Some of my workspaces have missing groups. That something that could happen, no ? eg issue #1025 ;)